docs/overview.rst

```diff
@@ -298,18 +298,23 @@ accessed by clicking the *View metrics* link at the top right.
 Cluster Backups
 ---------------
 
-On the Cluster Backups page you can see any backups of your cluster. The
-dropdown menu at the top left allows you to select the timespan of backups you
-want to see.
+You can find the Backups page in the detailed view of your cluster. By
+default, a backup is made every hour. You can see and restore all existing
+backups here. To restore a particular backup, simply click the *Restore*
```

Comment:
when the user clicks the restore backup button they see a modal window with a sql command they can run in the Admin UI. it's not done via the API yet, so it's not as simple as just clicking a button. But that doesn't mean the instructions here are wrong just that you might feel that more detailed instructions are needed, e.g. copy the sql command and run it in Admin UI